installing/installing_bare_metal_ipi/ipi-install-installation.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ipi-install-installation"]
+= Installing OpenShift
+include::_attributes/common-attributes.adoc[]
+:context: ipi-install-installation
+
+toc::[]
+
+// Deploying the cluster via the {product-title} installer
+include::modules/ipi-install-deploying-the-cluster-via-the-openshift-installer.adoc[leveloffset=+1]
+
+// Following the installation
+include::modules/ipi-install-following-the-installation.adoc[leveloffset=+1]
```

Comment:
Does this mean "After the installation" or "Following the progress of the installation"?

Reply:
I would consider rewording as well.